app/app-controllers/error-screen/errorController.js

```diff
@@ -73,24 +80,37 @@ define([
 
     ErrorController.prototype._generateErrorCallback = function(errorType, params) {
         var self = this;
+        var navigator = params.navigator;
         var backHandler = params.backHandler;
         var retryHandler = params.retryHandler;
         var isActiveItem = params.isActiveItem;
+        var canGoBack = params.canGoBack;
 
         return function(eventArgs) {
+            navigator.loaded = false;
+
             if (isActiveItem()) {
                 self.viewPlugin.once('back', function() {
                     self.hide();
                     self._removeModalEvents();
                     backHandler();
                 });
+
+                // Wait until the error page is loaded before showing
+                self.viewPlugin.on('error:loaded', function() {
```

Comment:
Hmm good idea @jer! eXtra does this for adaptive pages :) I will make that event a little more generic so we can reuse it :)